README.md

```diff
@@ -39,6 +39,66 @@ information you think necessary. Refer to [the Npcap Guide section on reporting
 bugs](https://npcap.com/guide/npcap-users-guide.html#npcap-issues) for more
 complete directions.
 
+## Contribution
+
+If you want to contribute to the development of this project, first you need to setup the development environment.
+
+### Development Environment
+
+Install a Windows. For example, use Windows 11, version 21H2. 
+
+When you install Visual Studio 2022, select the Desktop development with C++ workload, then under Individual Components add:
```

Comment:
"Install Visual Studio 2022" should explicitly be stated as a step, for example:

> Install Visual Studio 2022. When you do so, select the Desktop development with C++ workload, then under Individual Components add: